Sources/Tools/CMakeLists.txt

```diff
@@ -41,10 +41,10 @@ function(_add_tools)
 
     add_dependencies(tools ${_at_TOOLS})
     if(_at_GUI)
-        install(TARGETS ${_at_TOOLS} DESTINATION tools_gui)
+        install(TARGETS ${_at_TOOLS} OPTIONAL DESTINATION tools_gui)
         set(_TOOLS_GUI ${_at_TOOLS} ${_TOOLS_GUI} PARENT_SCOPE)
     else()
-        install(TARGETS ${_at_TOOLS} DESTINATION tools_cli)
+        install(TARGETS ${_at_TOOLS} OPTIONAL DESTINATION tools_cli)
```

Comment:
I can't imagine we ever want to actually install the benchmark program...  Perhaps it should just be a simple `add_subdirectory` instead of part of the tools target...

Reply:
The downside is that it then becomes more difficult to get to the benchmark program if you want it, especially on multi-config generators. Navigating the build tree to the generated artifacts is unpleasant IME. :/